server/src/main/java/org/elasticsearch/monitor/jvm/JvmGcMonitorService.java

```diff
@@ -165,13 +165,20 @@ public JvmGcMonitorService(Settings settings, ThreadPool threadPool) {
     }
 
     private static TimeValue getValidThreshold(Settings settings, String key, String level) {
-        TimeValue threshold = settings.getAsTime(level, null);
+        final TimeValue threshold;
+
+        try {
+            threshold = settings.getAsTime(level, null);
```

Comment:
Also, I'm wondering if the better long-term fix is to make the GC threshold settings not be group settings anymore, but affix settings. Then we would not need to resort this, as we'd get the proper setting named pushed down anyway, and this wrapping would be necessary. I think that would be a good follow up to this one.